channeldb/db.go

```diff
@@ -251,7 +255,10 @@ func createChannelDB(dbPath string) error {
 	}
 
 	path := filepath.Join(dbPath, dbName)
-	bdb, err := kvdb.Create(kvdb.BoltBackendName, path, true)
+	bboltOpts := &bbolt.Options{
+		NoFreelistSync: true,
```

Comment:
Though this is just for creating the db, should this be user-specified as well? Setting it one way or another shouldn't give any performance benefit as:
- if we sync the freelist (this takes time), the next startup upon `Open` will be shorter
- if we don't sync the freelist (quick `Create` time), the next startup upon `Open` will be longer as it needs to construct the freelist

So both scenarios are quicker in one area and slower in another. Only reason I think would be for consistency.